### DISCUSSION TITLE
Questions for the reset button in AdBlock settings

### DISCUSSION THREAD

**Discussion by zhmars** [bromite/bromite#1892] [General]
When I first clicked the button, what I expected was resetting the filter url to default (`https://www.bromite.org/filters/filters.dat`). But it behaves just like the `Cancel` button (reset to the last saved url). 

Should it been considered to be redundant? If so, could we change its behavior to `reset to default`?

**Comment by ghost** [comment]
The workaround:

put `https://www.bromite.org/filters/filters.dat` manually in filter url.

**Comment by csagan5** [comment]
I will change the behaviour to not close the activity after tapping the reset button.
